resources/roca_lib.brs

```diff
@@ -97,6 +98,16 @@ sub __xit(description as string, func as object)
     m.__suite.__registerCase("skip", description, m.__suite, func)
 end sub
 
+' Fields to add to `m` in the case context.
+' @param fields a roAssociativeObject
+sub __roca_addFields(fields as object)
```

Comment:
I think I'd prefer if we didn't call this `addFields` because of the overlap with `ifSGNodeDict#addFields` — I suspect one day I'm going to forget `m` isn't an instance of `roSGNode` and be very confused.

Reply:
Yes that's a fair point. I chooses to imitate the node method but it may be needlessly confusing

Reply:
Renamed